dpctl/tensor/_dlpack.pyx

```diff
@@ -965,41 +965,66 @@ cdef object _create_device(object device, object dl_device):
 def from_dlpack(x, /, *, device=None, copy=None):
     """from_dlpack(x, /, *, device=None, copy=None)
 
-    Constructs :class:`dpctl.tensor.usm_ndarray` instance from a Python
-    object ``x`` that implements ``__dlpack__`` protocol.
+    Constructs :class:`dpctl.tensor.usm_ndarray` or :class:`numpy.ndarray` instance
+    from a Python object ``x`` that implements ``__dlpack__`` protocol.
 
     Args:
         x (object):
             A Python object representing an array that supports
             ``__dlpack__`` protocol.
         device (Optional[str, :class:`dpctl.SyclDevice`, :class:`dpctl.SyclQueue`, :class:`dpctl.tensor.Device`, tuple([:class:`enum.IntEnum`, int])])):
-            Array API concept of a device where the output array is to be placed.
-            ``device`` can be ``None``, a oneAPI filter selector
-            string, an instance of :class:`dpctl.SyclDevice` corresponding to
-            a non-partitioned SYCL device, an instance of
-            :class:`dpctl.SyclQueue`, a :class:`dpctl.tensor.Device` object
-            returned by :attr:`dpctl.tensor.usm_ndarray.device`, or a
-            2-tuple matching the format of the output of the ``__dlpack_device__``
-            method, an integer enumerator representing the device type followed by
-            an integer representing the index of the device. The only supported
-            :class:`dpctl.tensor.DLDeviceType` types are "kDLCPU" and
-            "kDLOneAPI".
+            Device where the output array is to be placed. ``device`` keyword values can be:
+
+            * ``None``
+                The data remains on the same device. If the data backing up
+                input object ``x`` resides on ``"kDLCPU"`` device, the return
+                type would be :class:`numpy.ndarray`, otherwise the return
+                type would be :class:`dpctl.tensor.usm_ndarray`.
```

Comment:
```suggestion
                type will be :class:`numpy.ndarray`, otherwise the return
                type is :class:`dpctl.tensor.usm_ndarray`.
```